packages/fungible-token/bridge-fungible-token/tests/functions/bridge/mod.rs

```diff
@@ -115,7 +118,7 @@ mod success {
             message_receipt.recipient().unwrap()
         );
         assert_eq!(message_receipt.amount().unwrap(), 0);
-        assert_eq!(message_receipt.len().unwrap(), 104);
+        assert_eq!(message_receipt.len().unwrap(), 100);
```

Comment:
What changed about the len of the receipts?

Reply:
This is likely due to a correction in the serialized format of message receipts: https://github.com/FuelLabs/fuel-vm/commit/18a2e4a13094448068e378baf3b89e448ec356d6

Reply:
Ah it was also changed here: https://github.com/FuelLabs/fuel-bridge/pull/34/files#diff-73b34b7aa5b07f5607f9ebcf819a8d15226bb84d5553ca531373dbeb9104cabeR8